### DISCUSSION TITLE
Getting 404 when trying to request a review on a PR

### DISCUSSION THREAD

**Discussion by SimonBrandner** [community/community#11738] [Pull Requests]
When trying to request a review on a PR I get:

```
POST https://github.com/matrix-org/matrix-react-sdk/pull/5672/review-requests 404
```

![2022-02-18_21-26](https://user-images.githubusercontent.com/25768714/154757387-1bfb8d63-58e3-4a22-9056-750c4719df0e.png)

Repro steps:

1. Be a collaborator in a repo
2. Make a PR
3. Wait for a review from a team (of an organisation you're not a member of) to be requested
4. Try to (re)request a review from a specific person 
5. See an error in the console, the UI acts as if nothing happened

**Comment by egnor** [comment]
Same, though I certainly don't see this all the time, but it's consistent in the same PR, which is frustrating
![image](https://user-images.githubusercontent.com/279819/179432274-a7f22efd-c850-4730-b55e-ec75f91df4b9.png)

Also see: https://github.community/t/can-not-request-a-re-review-of-a-pull-request/244249
